### DISCUSSION TITLE
Motion detection lentissimo con camere Reolink (RLC-811A)

### DISCUSSION THREAD

**Discussion by paolomorabito** [indomus/forum#1498] [HUB: Home Assistant]
Ciao a tutti,
ho diverse camere Reolink (tra cui la 811A) che ho integrato su HA tramite https://github.com/fwestenberg/reolink_dev
Tutto funziona, ma il motion detection non è istantaneo, nel senso che il rilevamento del movimento mi compare su HA dopo tantissimi secondi (a volte un minuto). È normale tutto questo? C'è un modo per ricevere istantaneamente il segnale di allarme in maniera da creare automatismi su HA?

Grazie

**Comment by paolomorabito** [comment]
Mi rispondo da solo. Cambiando il parametro "Motion states update fallback delay (seconds)" dell'integrazione e mettendolo a 3 secondi, ho risolto il problema, anche se, per come indicato, potrebbe influenzare le prestazioni della camera. Attualmente, comunque, funziona con un ritardo, nella segnalazione del movimento, che può andare da1 a 5-6 secondi d media.

**Reply by LevisDomus** [reply]
Ciao Paolo, sono prossimo all'acquisto delle stesse telecamere le 811A (ben 3) più una coppia di Duo Floodlight PoE. Volevo chiederti un parere, se ha senso inserite queste camere sotto un NVR tipo quelle dea Reolink oppure farle scrivere direttamente su una NAS ed integrarle poi sotto HA. Tu come hai fatto? Sei in generale contento della loro prestazione? Grazie
